libraries.json

```
@@ -0,0 +1,16 @@
+{
+	"Angular": {
+		"imports": ["angular"],
```

I'd change this and deal with AngularJS later

```suggestion
		"imports": ["@angular/"],
```

---

Will do this in the next PR